tasks/php_apache.yml

```diff
@@ -1,8 +1,3 @@
-  - name: LAMP | FAST-CGI configuration for php
-    template: src={{role_dir}}/templates/apache2/conf/fastcgi.conf.j2 dest=/etc/apache2/conf-available/fastcgi.conf
-    when: "apache_mode == 'worker'"
-    become: yes
-    tags: setup
 
```

Comment:
^^^^ you moved this below enabling the conf. 

That's why automated test fails on step - on first run you don't have template created => nothing to enable.

TASK [role_under_test : LAMP | a2enconf fastcgi] *******************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["a2enconf", "fastcgi"], "delta": "0:00:00.065135", "end": "2017-03-06 10:23:47.813574", "failed": true, "rc": 1, "start": "2017-03-06 10:23:47.748439", "stderr": "ERROR: Conf fastcgi does not exist!", "stdout": "", "stdout_lines": [], "warnings": []}

Requested change: order needs to stay the same